src/reports/ChartReportPage.jsx

```diff
@@ -112,6 +115,21 @@ const ChartReportPage = () => {
             </Select>
           </FormControl>
         </div>
+        <div className={classes.filterItem}>
+          <FormControl fullWidth>
+            <InputLabel>{t('commandIndex')}</InputLabel>
```

Comment:
I don't think this is the right label. We should probably add a new string. Maybe it should just say "time".

Reply:
You caught me. You're right, it definitely should say "Time". I just couldn't find it in the existing strings and wasn't sure how to go about adding a new one (in multiple languages).

Reply:
Is there a trick to generating a string in all of the languages or should i go ahead and do it manually?

Reply:
You should only add English. Translations are handled through Transifex.

Reply:
> You should only add English. Translations are handled through Transifex.

Amazing, I avoided it for no reason 😆 
So I just add it to `src/resources/l10n/en.json` ?

Reply:
I think I did that right..